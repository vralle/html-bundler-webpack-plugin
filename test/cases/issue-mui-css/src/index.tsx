import { createRoot } from 'react-dom/client';

// test both together must work
import Stack from '@mui/material/Stack';
import './style.css';

const root = createRoot(document.getElementById('app')!);

root.render(
  <Stack>
    <h1>Hello World!</h1>
  </Stack>
);
