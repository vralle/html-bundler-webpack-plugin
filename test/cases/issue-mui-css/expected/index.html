<!DOCTYPE html>
<html>
  <head>
    <title>Test</title>
  <link href="index.dc4ea4af.css" rel="stylesheet">
</head>
  <body>
    <h1>Hello World!</h1>
    <div id="app"></div>
    <script src="index.f575b782.js"></script>
  </body>
</html>